app/models/carrier_profile.rb

```diff
@@ -54,6 +54,10 @@ def plans
     Plan.where(carrier_profile_id: self._id)
   end
 
+  def is_group_size?
```

Comment:
same as above. 

carrier 34484 already has anEmployerGroupSizeFactorSet in the DB with rating_factor_entries corresponding to each of the supported group size counts. 

That's the entire purpose of the lookup method on line 557 and 563